.github/workflows/doc-prod.workflow.yml

```diff
@@ -0,0 +1,38 @@
+name: Deployment Doc Prod
+
+on:
+  push:
+    branches:
+      - master
+      - 6-stable
+
+jobs:
+  doc-prod:
+    name: Deployment Doc Prod
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+      - uses: ./.github/actions/doc-deploy
+        env:
```

Comment:
Do not use environnement variables for your custom actions prefer the inputs/outputs system directly from your action file

Reply:
Ditto